finesse/gui/temp_control.py

```diff
@@ -0,0 +1,359 @@
+"""Panel and widgets related to temperature monitoring."""
+
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_qt5agg import FigureCanvasQTAgg
+from PySide6.QtCore import Qt
+from PySide6.QtGui import QPixmap
+from PySide6.QtWidgets import (
+    QGridLayout,
+    QGroupBox,
+    QLabel,
+    QLineEdit,
+    QPushButton,
+    QSlider,
+    QSpinBox,
+    QWidget,
+)
+
+# plt.style.use("./finesse_gui.style")
+plt.rcParams["figure.facecolor"] = "black"
+plt.rcParams["axes.facecolor"] = "black"
+plt.rcParams["axes.edgecolor"] = "white"
+plt.rcParams["text.color"] = "white"
+plt.rcParams["xtick.color"] = "white"
+plt.rcParams["ytick.color"] = "white"
```

Comment:
I'm not sure about setting the plot parameters globally like this. It's fine for now, but it might pose a problem later down the line if we want to produce another plot somewhere else in the code that looks different.

I guess you're trying to follow the colour scheme from the screen shot of the old program here? That's good, but tbh I think the `matplotlib` defaults are fine here, so I'd just drop this chunk. Personally I'm not a fan of black backgrounds for plots.

If @jonemurray really wants to keep this colour scheme though (or do something else...) we can always accommodate that.